spec/features/admin/admin_merch_dashboard_spec.rb

```diff
@@ -72,25 +75,48 @@
     within ".merchant-#{@mike.id}" do
       expect(page).to have_content("Merchant Name: #{@mike.name}")
       expect(page).to have_content("Status: disabled")
-      # expect(page).to_not have_button("Disable")
+      expect(page).to_not have_button("Disable")
     end
 
     within ".merchant-#{@meg.id}" do
       expect(page).to have_content("Merchant Name: #{@meg.name}")
       expect(page).to have_content("Status: enabled")
       click_button("Disable")
     end
+    @meg.reload
 
     expect(current_path).to eq("/admin/merchants")
     expect(page).to have_content("Merchant account has been disabled.")
 
+    expect(@meg.status).to eq("disabled")
+
     within ".merchant-#{@meg.id}" do
       expect(page).to have_content("Merchant Name: #{@meg.name}")
       expect(page).to have_content("Status: disabled")
       expect(page).to_not have_button("Disable")
     end
-
-    # expect(@meg.status).to eq("disabled")
   end
 
+  it "when a merchant is disabled all of their items are also disabled" do
+    expect(@meg.status).to eq ("enabled")
+    expect(@meg.items).to eq ([@tire, @seat, @pump])
+    expect(@tire.active?).to eq(true)
+    expect(@seat.active?).to eq(true)
+    expect(@pump.active?).to eq(true)
+
+    visit "/admin/merchants"
+
+    within ".merchant-#{@meg.id}" do
+      click_button("Disable")
+    end
+    @meg.reload
+    @tire.reload
+    @seat.reload
+    @pump.reload
+    expect(@meg.status).to eq ("disabled")
```

Comment:
mystery solved!  we so needed this `reload` . . . now we can trust our testing again!